snippets/capabilities-table.adoc

```diff
@@ -0,0 +1,22 @@
+:_content-type: SNIPPET
+
+The following table describes the `baselineCapabilitySet` values.
+
+.Cluster capabilities `baselineCapabilitySet` values description
+[cols=".^4,.^6a",options="header"]
+|===
+|Value|Description
+
+|`vCurrent`
+|Specify when you want to automatically add new capabilities as they become recommended.
+
+|`v4.11`
+|Specify when you want the capabilities defined in {product-title} 4.11 and not automatically enable capabilities, which might be introduced in later versions.
```

Comment:
nit, and this wording predates this pull request, but `v4.y` sets are canonically "[the recommended set of optional capabilities to enable for the 4.y][1]".  For 4.11 and 4.12, we decided to recommend all the capabilities defined in each 4.y.  But it's possible that 4.13 will add a new capability that we withhold from `v4.13`, so I'd recommend "recommended" instead of "defined" in your definitions here.

[1]: https://github.com/openshift/api/blob/a671aa80d9956a81ed6e17fa0d8a71fbc7a0fbc0/config/v1/types_cluster_version.go#L312-L322

Reply:
Thanks for pointing this out. I'll use "recommended" for accuracy